modules/atomic-widgets/prop-types/concerns/has-meta.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\PropTypes\Concerns;
+
+use Elementor\Modules\AtomicWidgets\PropTypes\Contracts\Prop_Type;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * @mixin Prop_Type
+ */
+trait Has_Meta {
+	protected array $meta = [];
+
+	public function meta( $key, $value = null ): self {
+		$is_tuple = is_array( $key ) && 2 === count( $key );
```

Comment:
what is the usage for tuple?

Reply:
```php
String_Prop_Type::make()->meta( Dyanmic_Prop_Type::ignore() );
```

the `ignore` method returns `['dynamic', false]` this allows more convenient way for widgets builders to disable dynamic on some prop type